2010-06-04  Michael Tautschnig  <mt@debian.org>

	* ainsl.1: Document ainsl exit codes (thanks Rob <rclemley@booksys.com>)
Index: trunk/man/ainsl.1
===================================================================
--- trunk.orig/man/ainsl.1
+++ trunk/man/ainsl.1	
@@ -28,6 +28,9 @@
 are used for matching, not when adding the line. Additionaly,
 the following characters are escaped in LINE: ( ) +
 
+The exit code returned by ainsl is 0 on success and non-zero
+on error.  See the section EXIT CODES.
+
 .SH OPTIONS
 .TP
 .B \-a
@@ -64,6 +67,17 @@
 
 Add exports entry without exactly matching spaces.
 
+.SH EXIT CODES
+.TP
+.B 0
+Success: Either FILE contains the LINE/PATTERN or LINE was appended to FILE.
+.TP
+.B 2
+FILE does not exist and the autocreate option (-a) is not present.
+.TP
+.B 13
+FILE cannot be opened for writing and LINE/PATTERN was not found in file.
+
 .SH NOTES
 This is a similar function to AppendIfNoSuchLine from cfengine(8).
 .SH SEE ALSO
